com.vscodium.codium-insiders.yaml

```diff
@@ -0,0 +1,144 @@
+id: com.vscodium.codium-insiders
+sdk: org.freedesktop.Sdk
+runtime: org.freedesktop.Sdk
+runtime-version: '23.08'
+command: codium-insiders
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --socket=ssh-auth
+  - --share=network
+  - --device=all
+  - --filesystem=host
+  - --filesystem=xdg-config/gtk-3.0:ro
+  - --persist=.vscode-oss
+  - --allow=devel
+  - --talk-name=org.freedesktop.secrets
+  - --talk-name=org.freedesktop.Flatpak
+  - --filesystem=xdg-run/gnupg:ro
+  - --filesystem=xdg-config/kdeglobals:ro
+  - --talk-name=com.canonical.AppMenu.Registrar
+  - --system-talk-name=org.freedesktop.login1
+separate-locales: false
+add-extensions:
+  com.visualstudio.code.tool:
+    directory: tools
+    subdirectories: true
+    version: '23.08'
+    add-ld-path: lib
+    no-autodownload: true
+cleanup:
+  - /include
+  - /lib/pkgconfig
+  - /share/gtk-doc
+  - '*.la'
+modules:
+  - shared-modules/libsecret/libsecret.json
+  - shared-modules/libusb/libusb.json
+  - name: codium-insiders
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 codium-insiders.svg /app/share/icons/hicolor/scalable/apps/com.vscodium.codium-insiders.svg
+      - |
+        for size in 16 32 48 64 128 256 512; do
+          rsvg-convert --width ${size} --keep-aspect-ratio --format png --output codium-insiders_${size}.png codium-insiders.svg || exit 1
+          install -Dm644 codium-insiders_${size}.png /app/share/icons/hicolor/${size}x${size}/apps/com.vscodium.codium-insiders.png || exit 1
+        done
+      - install -Dm644 com.vscodium.codium-insiders.metainfo.xml -t /app/share/metainfo
+      - ar x codium.deb
+      - tar xf data.tar.xz
+        # HACK: Use directory for IPC sockets that is shared between Flatpak instances
+        # Replace XDG_RUNTIME_DIR by XDG_RUNTIME_DIR/app/FLATPAK_ID
+      - find usr/share/codium-insiders -type f -print0 | xargs -0 sed -i "s/\bjoin([^)]*xdg_\?runtime_\?dir[^,)]*/\0,'app',process.env.FLATPAK_ID/gI"
+        # and disable integrity check
+      - sed -e 's/"checksums":/"_checksums":/' -i usr/share/codium-insiders/resources/app/product.json
+      - mv usr/share/codium-insiders /app/share
```

Comment:
I see a mime file in the deb that is not being installed. I think you need to install it in `/spp/share/mime/packages` and run `update-mime-database` for it to be useful.

Reply:
you just need to install it with the proper name `$FLATPAK_ID.xml`. Nothing else.